tests/api/test_groups.py

```diff
@@ -287,13 +295,13 @@ def test_GroupsMixin_update_concession_group_funding_source_expiry(
     mock_ClientProtocol_put_update_concession_group_funding_source, ListResponse_GroupFundingSources, mocker
 ):
     client = GroupsMixin()
-    mocker.patch.object(client, "_format_concession_expiry", return_value="formatted concession expiry")
+    mocker.patch.object(client, "_format_expiry", return_value="formatted concession expiry")
```

Comment:
Same nit about `concession`

Reply:
Removed in https://github.com/cal-itp/littlepay/pull/45/commits/484cf53ab421a1254e28da48bb39364e141b8f8c